cosmetics-web/app/models/nano_element.rb

```diff
@@ -1,6 +1,15 @@
 class NanoElement < ApplicationRecord
   belongs_to :nano_material
 
+  # TODO: add uniquenes notification across notifications
```

Comment:
If I infer correctly the intention of the comment:
```suggestion
  # TODO: add uniquenes validation across notifications
```